algorithme-evaluation.md

```diff
@@ -90,9 +93,70 @@ Le modèle est entrainé sur les variables d'apprentissage suivantes:
+
+Score AUCPR
+: l'aire sous la courbe rappel-précision (Area Under Curve, for Precision-Recall curve). C'est une courbe obtenue à partir du rappel en abscisse, et de la précision en ordonnée, et qui permet d'étudier la performance du modèle en fonction du seuil de classification choisi
+
+Pour plus d'informations sur ces métriques, voir les liens ci-dessous:
+
+- [Précision et rappel](https://fr.wikipedia.org/wiki/Pr%C3%A9cision_et_rappel)
+- [Matrice de confusion](https://fr.wikipedia.org/wiki/Matrice_de_confusion)
+- [Score $F_{\beta}$](https://en.wikipedia.org/wiki/F-score)
+
+### _Seuils de détection_
+
+Le premier étage algorithmique produit un score de risque de défaillance, compris entre 0 (aucun signal de risque) et 1 (risque fort détecté).
+A partir de ces scores de risque, une liste d'entreprise à risque est construite, avec trois palliers de risques:
+
+- un niveau "risque fort" :red_circle: où la précision est élevée, c'est-à-dire que les entreprises identifiées comme à risque fort le sont effectivement, quitte à manquer quelques entreprises qui font défaillance
+- un niveau "risque modéré" :orange_circle: est construite de sorte à capturer un maximum d'entreprises à risque, quitte à avoir dans cette liste plus de faux positifs, c'est-à-dire d'établissements qui sont en réalité en bonne santé.
+- un niveau "aucun signal de risque" :green_circle:, comprenant tous les établissements de notre périmètre n'entrant pas dans les deux palliers ci-dessus.
+
+Ces seuils sont déterminés par la maximisation du score $F_{\beta}$, une métrique permettant de prendre en compte à la fois les faux positifs et les faux négatifs.
+Plus particulièrement:
+
+- le seuil du pallier "risque fort" est choisi pour maximiser le $F_{0.5}$, une métrique qui favorise deux fois plus la précision que le rappel. Ce score favorise ainsi une précision élevée, et donc l'exclusivité d'établissements effectivement en défaillance dans le pallier "risque fort".
+- le seuil du pallier "risque modéré" est choisi pour maximiser le score $F_2$, qui favorise deux fois plus le rappel que la précision. La maximisation de cette métrique vise à obtenir un palleir "risqe modéré" qui capture un maximum d'établissements effectivement en défaillance, quitte à capturer "par erreur" des faux positifs, c'est-à-dire quitte à viser trop large et lister des entreprises qui n'entreront pas en défaillance.
```

Comment:
```suggestion
- le seuil du pallier "risque modéré" est choisi pour maximiser le score $F_2$, qui favorise deux fois plus le rappel que la précision. La maximisation de cette métrique vise à obtenir un pallier "risque modéré" qui capture un maximum d'établissements effectivement en défaillance, quitte à capturer "par erreur" des faux positifs, c'est-à-dire quitte à viser trop large et lister des entreprises qui n'entreront pas en défaillance.
```

Reply:
Also fixed in https://github.com/signaux-faibles/documentation/pull/45/commits/aca623779b339ee2d603892b966e652b5a56d659